lib/git-shell-out-strategy.js

```diff
@@ -80,6 +85,12 @@ export default class GitShellOutStrategy {
         PATH: process.env.PATH || '',
       };
 
+      ENV_VARS_TO_COPY.forEach(envVar => {
+        if (process.env[envVar] !== undefined) {
+          env[envVar] = process.env[envVar];
+        }
+      });
+
```

Comment:
I feel like I've done this a few times now. It might be worth refactoring into a helper somewhere 🤔